extensions/amp-jwplayer/1.0/amp-jwplayer.js

```diff
@@ -0,0 +1,117 @@
+import {getDataParamsFromAttributes} from '#core/dom';
+import {dict} from '#core/types/object';
+
+import {isExperimentOn} from '#experiments';
+
+import {userAssert} from '#utils/log';
+
+import {BaseElement} from './base-element';
+
+import {CSS} from '../../../build/amp-jwplayer-1.0.css';
+import {
+  getConsentMetadata,
+  getConsentPolicyInfo,
+  getConsentPolicyState,
+} from '../../../src/consent';
+
+/** @const {string} */
+const TAG = 'amp-jwplayer';
+
+class AmpJwplayer extends BaseElement {
+  /** @override */
+  init() {
+    super.init();
```

Comment:
Is this call necessary?

Reply:
i think it needs to call super.init() to register the video actions, brightcove player does it as well